drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -96,6 +96,37 @@ static void syscmd_status_not(SHCI_TL_CmdStatus_t status)
 	BT_DBG("status:%d", status);
 }
 
+static void tryfix_event(TL_Evt_t *tev)
+{
+/*
+ * https://github.com/zephyrproject-rtos/zephyr/issues/19509
+ * Tested on nucleo_wb55rg (stm32wb55rg) BLE stack (v1.2.0)
+ * Unresolved Resolvable Private Addresses (RPA)
+ * is reported in the peer_rpa field, and not in the peer address,
+ * as it should, when this happens the peer address is set to all FFs
+ * 0A 00 01 08 01 01 FF FF FF FF FF FF 00 00 00 00 00 00 0C AA C5 B3 3D 6B ...
+ * If such message is passed to HCI core than pairing will essentially fail.
+ * Solution: Rewrite the event with the RPA in the PEER address field
+ */
+	struct bt_hci_evt_le_meta_event *mev =
+			(struct bt_hci_evt_le_meta_event *)&tev->payload;
+	u8_t *ptr = ((u8_t *)mev + (sizeof(*mev)));
+
+	if (tev->evtcode == BT_HCI_EVT_LE_META_EVENT &&
+			mev->subevent == BT_HCI_EVT_LE_ENH_CONN_COMPLETE) {
```

Comment:
This should be aligned with the opening ( on the first line

Reply:
Oh, it even looks better. Thanks. Done.